src/pyscipopt/scip.pyx

```diff
@@ -4537,6 +4537,15 @@ cdef class Model:
 
         free(_coeffs)
 
+    def chgVarBranchPriority(self, Variable var, priority):
+        """Set the branch priority of a variable.
```

Comment:
can you copy verbose the documentation from scip, please:

> sets the branch priority of the variable; variables with higher branch priority are always preferred to variables with lower priority in selection of branching variable

Reply:
Thank you for your comment. I changed it.